src/Symfony/Component/Message/Debug/LoggingMiddleware.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Message\Debug;
+
+use Symfony\Component\Message\MiddlewareInterface;
+use Psr\Log\LoggerInterface;
+
+/**
+ * @author Samuel Roze <samuel.roze@gmail.com>
+ */
+class LoggingMiddleware implements MiddlewareInterface
+{
+    /**
+     * @var LoggerInterface
+     */
```

Comment:
we don't document properties that can only be set at construction as they are autocompleted thanks to the typehint present in the constructor signature